.github/workflows/man-page-build-install.yml

```
@@ -0,0 +1,72 @@
+name: Man Page Build and Install
+
+on:
+  push:
+  pull_request:
+
```

We can prevent writes to the repos. An extra security thing.

```suggestion

permissions:
  contents: read

```

---

Is this worth adding everywhere? Outside of a couple of our automation workflows, everything does reads.

---

Yeah, I guess it's from some list of best-practices. Maybe OpenSSF? I can't see it now but this list has a lot of ideas: https://best.openssf.org/SCM-BestPractices/github/